src/uct/ib/rc/base/rc_ep.h

```diff
@@ -353,7 +351,8 @@ uct_rc_txqp_completion_inl_resp(uct_rc_txqp_t *txqp, const void *resp, uint16_t
 static UCS_F_ALWAYS_INLINE uint8_t
 uct_rc_iface_tx_moderation(uct_rc_iface_t *iface, uct_rc_txqp_t *txqp, uint8_t flag)
 {
-    return (txqp->unsignaled >= iface->config.tx_moderation) ? flag : 0;
+    return ((txqp->unsignaled >= iface->config.tx_moderation) ||
```

Comment:
how about this optimization:
add variable like iface->tx.real_moderation, and make it so iface->tx.real_moderation = min(iface->config.tx_moderation, iface->tx.cq_available)
then, would add overhead only when CQE is used, which is ~1/64 of the times. instead of adding overhead for every send